lib_cxng/include/lcx_groestl.h

```diff
@@ -85,6 +85,8 @@ size_t cx_groestl_get_output_size(const cx_groestl_t *ctx);
  *
  * @param[in]  size Length of the digest.
  *
+ * @return          GROESTL identifier.
+ *
```

Comment:
this function is void, so it don't return anything?

Reply:
It is from the revert, I prefer to no update anything.